comp110_architecture.tex

```diff
@@ -0,0 +1,73 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Convexity-Based Collision Detection Algorithms for a 2D Game Engine}
+
+% Please do not change the subtitle
+\subtitle{COMP110 - Computer Architecture Essay}
+
+% Please put your student number in the author field
+\author{1507866}
+
+\begin{document}
+
+\maketitle
+
+\abstract{Please include an abstract of at most 100 words (these do not count towards your word count).}
+
+\section{Introduction}
+
+The purpose of collision detection is to check if objects are currently or going to intersect. In the application of video games this is vital for the ensuring that objects interact correctly so the game can be played as it was intended.  The intention of this essay is to analyse three convexity-based collision detection algorithms for use in a 2D game engine. The first algorithm is the Gilbert- Johnson–Keerth algorithm first proposed in 1988. The second algorithm uses bounding boxes and was proposed by Guo. The third is by Sulaiman et al and uses vectors to calculate whether objects are intersecting. 
```

Comment:
"as it was intended" -- could be more precise.

"The intention of" -- "This essay analyses"

"2d game engine" -- could be a more specific application context.

Make list consistent in style. If you are including authors and years, you should do so for all three algorithms.